zebrad/src/utils.rs

```diff
@@ -0,0 +1,5 @@
+//! Helper utilities for development.
```

Comment:
Doc: Can we be a bit more specific here?

```suggestion
//! Extensions used in [`Future`]s and async code.
```